test/source/tests/setup.ts

```diff
@@ -397,6 +397,14 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
       await attesterFrame.waitAndRespondToModal('error', 'confirm', 'Disallowed by your organisation rules');
     }));
 
+    ava.default('user@no-submit-org-rule.flowcrypt.com - do not submit to attester on key generation', testWithBrowser(undefined, async (t, browser) => {
```

Comment:
The test somehow didn't manage to reproduce the problematic behavior. Could you please try a scenario when the private key is already present on EKM + there is a no-submit rule?

Reply:
Sorry - maybe I have you wrong feedback! It seems you were actually able to reproduce the problem?

If so, then you could remove lines 403 and 404, because the line above is already testing for the successful setup (the modal pupped up before the setup was done).

Reply:
No, it didn't reproduce in this setting. I just had to figure out how to run a key generation test

Reply:
The problem was that `SetupKeyManagerAutogenModule` set the option `submit_main: true`